From b5e8a5bae55b40fe6c16ed57cf3d2caf8b36bb69 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 30 Jun 2020 15:16:22 +0800
Subject: [PATCH] drm/rockchip: EXPORT_SYMBOL(rockchip_drm_crtc_send_mcu_cmd)

rockchip_drm_crtc_send_mcu_cmd will be used by panel-simple.ko

Change-Id: I463e52335244564b5a837ff045cff0ab005ea25d
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/rockchip/rockchip_drm_drv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_drv.c b/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
index b4c199942387..ade63558ab85 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
@@ -1031,6 +1031,7 @@ int rockchip_drm_crtc_send_mcu_cmd(struct drm_device *drm_dev,
 
 	return 0;
 }
+EXPORT_SYMBOL(rockchip_drm_crtc_send_mcu_cmd);
 
 /*
  * Attach a (component) device to the shared drm dma mapping from master drm
-- 
2.35.3

